controllers/noderefutil/providerid.go

```diff
@@ -25,14 +27,17 @@ import (
 
 var (
 	// ErrEmptyProviderID means that the provider id is empty.
+	// Deprecated: This var has no function and is going to be removed in a future release.
```

Comment:
```suggestion
	// Deprecated: This var is going to be removed in a future release.
```
nit: "no function" is not really true, maybe let's just keep it simple

Reply:
cool